wordle-web/components/wordList.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-dialog max-width="500">
+    <template v-slot:activator="{ props: activatorProps }">
+      <v-btn
```

Comment:
Again, maybe add some space between this button and GUESS!. An idea could be to use a v-bottom-sheet for the valid words button so its separated, we did this!